simplipy/system/__init__.py

```diff
@@ -197,6 +198,42 @@ def alarm_going_off(self) -> bool:
         """
         return self._location_info["system"]["isAlarming"]
 
+    @property  # type: ignore
+    def cameras(self) -> Dict[str, Camera]:
+        """Return list of cameras.
+
+        :rtype: ``List[:meth:`simplipy.camera.Camera`]``
+        """
+        cameras = {}
+        for camera in self._location_info["system"]["cameras"]:
+            if camera["model"] != DOORBELL_MODEL:
+                cameras[camera["uuid"]] = Camera(
+                    self._request,
+                    self._get_entities,
+                    self.system_id,
+                    EntityTypes.camera,
+                    camera,
+                )
+        return cameras
+
+    @property  # type: ignore
+    def doorbells(self) -> Dict[str, Camera]:
+        """Return list of doorbells.
+
+        :rtype: ``List[:meth:`simplipy.camera.Camera`]``
+        """
+        doorbells = {}
+        for camera in self._location_info["system"]["cameras"]:
+            if camera["model"] == DOORBELL_MODEL:
+                doorbells[camera["uuid"]] = Camera(
+                    self._request,
+                    self._get_entities,
+                    self.system_id,
+                    EntityTypes.camera,
+                    camera,
+                )
+        return doorbells
+
```

Comment:
These two blocks of code are now nearly identical (except for a one-line difference). If for our purposes a doorbell is merely a type of camera, why not simply retain all of them as `system.cameras`?

Reply:
Because it may be relevant for other systems (e.g. HA) to know which one is which... Doorbells require to be registered differently with HomeKit, for instance.

Reply:
Understood. Nevertheless, we're still duplicating code. Can you think on how to combine these two blocks?

Reply:
@nzapponi I took a look at the latest and this one still seems really convoluted.

Reply:
What about now?

Reply:
I still think it's duplicated. Why do we need properties for both cameras and doorbells when the `camera_type` attribute can tell us which one we have?